docker-compose.yml

```diff
@@ -8,7 +8,6 @@ services:
   web:
     build: .
     command: python3 manage.py runserver 0.0.0.0:8000
-    command: python3 manage.py test
```

Comment:
This was giving me trouble when debugging - when your unit tests fail and you're trying to debug the API with Postman, this will prevent the server for starting (upon failing tests).
Have you tried using the droptoconsole.bash script to run unit tests? Once you run it, you can just execute "python3 manage.py test" to run tests.